freemocap/gui/qt/widgets/active_recording_widget.py

```diff
@@ -57,8 +57,13 @@ def get_active_recording_info(self, return_path: bool = False) -> Union[Recordin
 
     def set_active_recording(
         self,
-        recording_folder_path: Union[str, Path],
+        recording_folder_path: Union[str, Path, None],
     ):
+        if recording_folder_path is None or recording_folder_path == "None":
```

Comment:
Is it possible to change whatever is calling this so that if the parameter is `None` then `...set_active_recordin()` isn't called at all?

Reply:
It's connected directly to a signal in `main_window`, so we could add a `handle_X_signal` function in main window for that. As it is now, it seems better to handle the None type once in the function, vs. 8 separate times in the `main_window` function. Also, as I'll mention in the other conversation, I think setting the active recording to None is the best way of clearing it.